help-woke.rst

```diff
@@ -0,0 +1,46 @@
+:orphan:
```

Comment:
Please don't use orphans, but include the file in the navigation. There's no reason to hide it.

Reply:
I can do this if you feel strongly about it but the resource is intended to be accessed in the context of the README. It's a sub-page.

Reply:
Yes, it can be a subpage to the readme. But I feel pretty strongly about it that all pages should always be accessible from the navigation tree - otherwise it's very easy to miss that there is another page with information.